master/setup.py

```diff
@@ -203,6 +203,7 @@ def make_release_tree(self, base_dir, files):
         'psutil == 4.3.0',
         'jsonschema == 2.6.0',
         'freezegun == 0.3.9',
+        'names == 0.3.0',
```

Comment:
This should be a development requirement, not runtime (as long as you are careful with the imports).

Perhaps remove it from `setup.py` and create a `requirements-dev.txt` in the repo?